package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -24,6 +30,21 @@ def get_db(session_class: sessionmaker) -> Generator:
         database.close()
 
 
+def _get_dbname():
+    try:
+        return f"{getpass.getuser()}.db"
+
+    except Exception as exc:  # pylint: disable=broad-except
+        unique_id = uuid.uuid4()
+        logger.warning(
+            """Something went wrong with getting the username. Generated unique id %s for 
+                 for user's database name  Exception: %s""",
+            unique_id,
+            exc,
+        )
+        return f"{unique_id}.db"
```

Comment:
Not sure whether making a unique id here is probably not a good idea because it gets re-generated every time you run kedro-viz so you might end up with lots of files on s3 🤔 It might be better to just suggest here that people set the environment variable `KEDRO_SQLITE_STORE_USERNAME` in this case. Not sure though, let's think about it.